pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -76,20 +85,33 @@ def test_dotted_column_names(self, engine, connection):
     def test_dotted_column_names_raw(self, engine, connection):
         """When Hive returns a dotted column name, and raw mode is on, nothing should be modified.
         """
-        row = connection.execution_options(hive_raw_colnames=True) \
-            .execute('SELECT * FROM one_row').fetchone()
+        row = connection.execution_options(hive_raw_colnames=True).execute(text('SELECT * FROM one_row')).fetchone()
+        
+        if sqlalchemy_version >= 1.4:
+            row = row._mapping
+
         assert row.keys() == ['one_row.number_of_rows']
         assert 'number_of_rows' not in row
         assert getattr(row, 'one_row.number_of_rows') == 1
         assert row['one_row.number_of_rows'] == 1
 
+    @with_engine_connection
+    def test_reflect_no_such_table(self, engine, connection):
```

Comment:
Created this override since Hive was throwing OperationalError with SQLAlchemy 1.3, 1.4, and 2.0 when both table and schema do not exists.